mutable_tree.go

```diff
@@ -31,7 +30,7 @@ var ErrVersionDoesNotExist = errors.New("version does not exist")
 type MutableTree struct {
 	*ImmutableTree                                     // The current, working tree.
 	lastSaved                *ImmutableTree            // The most recently saved tree.
-	orphans                  map[string]int64          // Nodes removed by changes to working tree.
+	orphans                  map[int64]int64           // Nodes removed by changes to working tree.
```

Comment:
Why do we still need orphans? Are there plans to remove them in a subsequent PR?

Reply:
the idea is to remove them, this wasn't done in this pr